pillar.example.sls

```diff
@@ -1,68 +1,93 @@
 postgresql:
   lookup:
+    repo:
```

Comment:
The thing is we have tens of formulas all doing their own repo management having duplicated code that need to be maintained to keep support for deb/rhel/etc.. As of https://github.com/bechtoldt/saltstack-repos-formula/blob/master/meta.yaml#L9 I plan to extend the repos formula to detect OS automatically. I think it is reasonable to repo mgmt to this formula. IMHO there are some _rare_ cases where external dependencies can be acceptable (user mgmt, repo mgmt). 
What do you think?